client/my-sites/site-settings/section-export.jsx

```diff
@@ -7,10 +7,25 @@ import { Provider } from 'react-redux';
 /**
  * Internal dependencies
  */
+import EmptyContent from 'components/empty-content';
 import ExporterContainer from 'my-sites/exporter';
+import i18n from 'lib/mixins/i18n';
 
 export default class SiteSettingsExport extends Component {
 	render() {
+		if ( this.props.site.jetpack ) {
+			return (
+				<EmptyContent
+					illustration="/calypso/images/drake/drake-jetpack.svg"
+					title={ i18n.translate( 'Want to export your site?' ) }
+					line={ i18n.translate( 'Visit your site\'s wp-admin for all your import and export needs.' ) }
```

Comment:
I copied this from [import](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/section-import.jsx#L207) - that string's text is the same so should we change it there too or leave it as is in both locations?
cc @dmsnell